docs/en/ingest-management/faq.asciidoc

```diff
@@ -227,3 +226,27 @@ and port for your setup. If you run everything locally, the address is
 you can copy the {es} endpoint URL from the overview page of your deployment.
 If you're not running in {ecloud}, make sure the {kib} and {es} HTTPS ports
 are both accessible; by default these are `5601` and `9200` respectively.
+
+[discrete]
+[[how-do-i-reset-a-missing-dashboard-asset]]
+== If I delete an integration dashboard asset from {kib}, how do I get it back?
+
+To reinstall the assets for a specific integration, you can use the API.
```

Comment:
```suggestion
To reinstall the assets for a specific integration, you can use the UI for most cases
and the {fleet} API for those not allowed in the UI.
```

Reply:
I think that's OK, but I wonder if we should mention where in the UI. We haven't really documented the Integrations UI very well yet. Maybe that's just something we need to add later?

Reply:
I'll submit that the move to make Integrations a top level (left hand) Kibana navigation item means that everyone in the world should find it.  We can leave it out for now and if we are proven wrong, then we can add more pointers in for it.